lib/ex_doc/formatter/epub/templates/extra_template.eex

```diff
@@ -1,5 +1,8 @@
 <%= head_template(config, %{title: title}) %>
     <%= content %>
+    <%# Extra content needed by the current markdown processor (i.e. custom CSS) %>
+    <%= ExDoc.Markdown.get_markdown_processor().before_closing_body_tag(:epub) %>
+    <%# Extra content specified by the user (i.g. custom CSS) %>
```

Comment:
You probably meant "i.e."

Reply:
Actually it was supposed to be "e.g.". The previous "i.e." should be changed to "e.g.". It's an example, not a clarification.